docs/source/intro-clx-predictive-maintenance.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>is "imon" a typo here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Ah, good catch. Thank you! :)</p>